authz/README.md

```diff
@@ -174,29 +174,152 @@ will be able to call any of the `gNSI.ssh` RPCs.
 }
 ```
 
-# OpenConfig Extension for the gMNI gRPC-based Authorization Policy telemetry
+## Managing the gRPC-based Authorization Policy
+
+### Initial (fresh out of the box) state assumption
+
+When a device boots for the first time it should have:
+
+1. The `gNSI.authz` service up and running
+1. A default gRPC-level Authorization Policy for every active gRPC service.
+
+   The initial default gRPC-level Authorization Policy can either allow access
+   to all RPCs or deny access to all RPCs except for the `gNSI.authz` family of
+   RPCs.
+
+### Updating the policy
+
+Every policy needs changes from time to time and the `gNSI.authz.Rotate()` RPC
+is designed to do this task.
+
+There are 5 steps in the process of updating (rotating) an gRPC-level
+Authorization Policy, namely:
+
+1. Starting the `gNSI.authz.Rotate()` streaming RPC.
+
+   As the result a streaming connection is created between the server (the
+   switch) and the client (the management application) that is used in the
+   following steps.
+
+   > **⚠ Warning**
+   > Only one `gNSI.authz.Rotate()` can be in progress.
```

Comment:
the auth policy is "declarative" for all servers and services on the device
All policies must be defined as previous policies will be replaced during the rotate

Reply:
done.